releases/release-4.0.6.md

```diff
@@ -0,0 +1,204 @@
+    - Fix different plans between select and update query [#19403](https://github.com/pingcap/tidb/pull/19403)
+    - Fix a data race for tiflash work index in region cache [#19362](https://github.com/pingcap/tidb/pull/19362)
+    - Fix logarithm functions [#19291](https://github.com/pingcap/tidb/pull/19291)
+    - Fix a unexpected error when TiDB spilling to disk [#19272](https://github.com/pingcap/tidb/pull/19272)
+    - Make single partition table support index join on the inner side [#19197](https://github.com/pingcap/tidb/pull/19197)
+    - Fix the wrong hash key for decimal [#19188](https://github.com/pingcap/tidb/pull/19188)
+    - Fix an issue that TiDB will raise no-regions error when table endKey and region endKey are same [#19895](https://github.com/pingcap/tidb/pull/19895)
+    - Fix an issue that unexpect sucess to alter partition [#19891](https://github.com/pingcap/tidb/pull/19891)
+    - Fix a wrong default max allowed packet for push downed expression [#19876](https://github.com/pingcap/tidb/pull/19876)
+    - Fix a wrong behavior for `Max/Min` on `ENUM/SET` column [#19869](https://github.com/pingcap/tidb/pull/19869)
+    - Fix read failure from system tables `tiflash_segments` and `tiflash_tables` when some tiflash node is down [#19748](https://github.com/pingcap/tidb/pull/19748)
+    - Fix a wrong result for aggregation `Count(col)` [#19628](https://github.com/pingcap/tidb/pull/19628)
+    - Fix a runtime error in `TRUNCATE` [#19445](https://github.com/pingcap/tidb/pull/19445)
+    - Fix `PREPARE` statement FROM `@Var` will fail when `Var` contains uppercase characters [#19378](https://github.com/pingcap/tidb/pull/19378)
+    - Fix a bug that schema charset modification in an uppercase schema will cause panic [#19302](https://github.com/pingcap/tidb/pull/19302)
+    - Fix a inconsistence of plan between `information_schema.statements_summary` and `explain`, which contains tikv/tiflash info [#19159](https://github.com/pingcap/tidb/pull/19159)
+    - Fix the file exists errors in tests for `select into outfile` [#19725](https://github.com/pingcap/tidb/pull/19725)
+
++ TiKV
+
+    - Fix the estimation error for a non-index column with collation enabled. [#8620](https://github.com/tikv/tikv/pull/8620)
+    - Fix the issue that green GC may miss locks during transferring regions. [#8460](https://github.com/tikv/tikv/pull/8460)
+    - Fix the CDC incorrect resolved TS timeout. [#8573](https://github.com/tikv/tikv/pull/8573)
+    - Fix a panic issue if a TiKV runs very slow during conf change. [#8497](https://github.com/tikv/tikv/pull/8497)
+    - Fix the deadlock between the PD client thread and other threads calling PD sync requests. [#8612](https://github.com/tikv/tikv/pull/8612)
+    - Update jemalloc to 5.2.1 to address memory allocation problem in huge page. [#8463](https://github.com/tikv/tikv/pull/8463)
+    - Fix unified thread pool hang for long running query. [#8427](https://github.com/tikv/tikv/pull/8427)
+
```

Comment:
```suggestion
+ PD

    - Add `initial-cluster-token` configuration to avoid different clusters communicate during bootstrap [#2922](https://github.com/pingcap/pd/pull/2922)
    - Fix the unit of store limit rate when the mode is auto [#2826](https://github.com/pingcap/pd/pull/2826)
    - Fix the issue that some scheduler persist config without solving error [#2818](https://github.com/tikv/pd/pull/2818)
    - fix empty http response in scheduler [#2871](https://github.com/tikv/pd/pull/2871) [#2874](https://github.com/tikv/pd/pull/2874)
```